iam_migrations/freeipa/src/main.rs

```diff
@@ -177,39 +179,50 @@ async fn driver_main(opt: Opt) {
 
         // Block on signals now.
         loop {
-            tokio::select! {
-                Ok(()) = tokio::signal::ctrl_c() => {
-                    break
-                }
-                Some(()) = async move {
-                    let sigterm = tokio::signal::unix::SignalKind::terminate();
-                    tokio::signal::unix::signal(sigterm).unwrap().recv().await
-                } => {
-                    break
-                }
-                Some(()) = async move {
-                    let sigterm = tokio::signal::unix::SignalKind::alarm();
-                    tokio::signal::unix::signal(sigterm).unwrap().recv().await
-                } => {
-                    // Ignore
-                }
-                Some(()) = async move {
-                    let sigterm = tokio::signal::unix::SignalKind::hangup();
-                    tokio::signal::unix::signal(sigterm).unwrap().recv().await
-                } => {
-                    // Ignore
-                }
-                Some(()) = async move {
-                    let sigterm = tokio::signal::unix::SignalKind::user_defined1();
-                    tokio::signal::unix::signal(sigterm).unwrap().recv().await
-                } => {
-                    // Ignore
+            #[cfg(target_family = "unix")]
```

Comment:
I wonder if we need to move this signal pattern into the junk draw given it's used in a few places.

Reply:
or maybe some kind of builder/macro thing, since there's a few different versions of it?